include/dt-bindings/gpio/gpio.h

```diff
@@ -7,6 +7,8 @@
 #ifndef ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_GPIO_H_
 #define ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_GPIO_H_
 
+#include <drivers/gpio_flags.h>
```

Comment:
So I have been wanting a similar change to Zephyr, but I think we need to get a more general concensus on making this change. The `gpio_dt_flags_t` is only 8-bits, so it does seem by design. We should open a discussion about this

Reply:
This is indeed by design: Zephyr only allows devicetree use of the specifier flags that are supported in devicetree files by Linux.  I believe the intent is to retain devicetree source file compatibility independent of OS.  Best to open an RFC issue if you want that changed.

Reply:
ACK, i will file a RFC

Reply:
submitted an RFC, I don't have access to add people 
https://github.com/zephyrproject-rtos/zephyr/issues/29908